src/java.base/unix/classes/sun/nio/fs/UnixFileAttributes.java

```diff
@@ -163,7 +164,7 @@ public FileTime lastAccessTime() {
 
     @Override
     public FileTime creationTime() {
-        if (UnixNativeDispatcher.birthtimeSupported()) {
+        if (UnixNativeDispatcher.birthtimeSupported() && !birthtime_invalid) {
```

Comment:
If you invert and rename this from "invalid" to "valid" then it would improve the readability.

Reply:
I agree as long as it initialized tp `true`.